CONTRIBUTING.md

```diff
@@ -8,6 +8,26 @@ The fastest way to get started is by running the Tembo docker image, where PGMQ
 docker run -d --name postgres -e POSTGRES_PASSWORD=postgres -p 5432:5432 quay.io/tembo/pgmq-pg:latest
 ```
 
+## PGXN Installation
+
+PGXN is a centralized network to make it easier to install and manage PostgreSQL extensions.
+
+You can install PGMQ directly using PGXN, the PostgreSQL Extension Network, which is a distribution system for open-source PostgreSQL extension libraries.
+
+Install PGMQ using PGXN
+
+```bash
+pgxn install pgmq
```

Comment:
Please add hyperlinks to pgxn, either link directly to the pgxn install instructions or put the instructions here.